scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -1561,24 +1577,49 @@ public Builder setTrustCertificateTypes(CertificateType... certificateTypes) {
 		}
 
 		/**
-		 * Set maximum number of deferred processed application data messages.
+		 * Set maximum number of deferred processed outgoing application data
+		 * messages.
 		 * 
-		 * Application data messages received or sent during a handshake may be
-		 * dropped or processed deferred after the handshake. Set this to limit
-		 * the maximum number of messages, which are intended to be processed
-		 * deferred. If more messages are sent or received, theses messages are
-		 * dropped.
+		 * Application data messages sent during a handshake may be dropped or
+		 * processed deferred after the handshake. Set this to limit the maximum
+		 * number of messages, which are intended to be processed deferred. If
+		 * more messages are sent, these messages are dropped.
 		 * 
-		 * @param maxDeferredProcessedApplicationDataMessages maximum number of
-		 *            deferred processed messages
+		 * @param maxDeferredProcessedOutgoingApplicationDataMessages maximum
+		 *            number of deferred processed messages
 		 * @return this builder for command chaining.
 		 * @throws IllegalArgumentException if the given limit is &lt; 0.
 		 */
-		public Builder setMaxDeferredProcessedApplicationDataMessages(final int maxDeferredProcessedApplicationDataMessages) {
-			if (maxDeferredProcessedApplicationDataMessages < 0) {
-				throw new IllegalArgumentException("Max deferred processed application data messages must not be negative!");
+		public Builder setMaxDeferredProcessedOutgiongApplicationDataMessages(
+				final int maxDeferredProcessedOutgoingApplicationDataMessages) {
```

Comment:
If we don't need the final keyword, could we remove it ?

Reply:
OK